### PR TITLE
Fix for restoring schema cleaning

### PR DESCRIPTION
**Pull request by jacekszubert**
- `LocalSecondaryIndexes` added
- check if `ProvisionedThroughput` key exists in index added
- `clean_table_schema` enabled to run always

### CODE REVIEW

File: dynamodb-bnr.py [hunk 1]
```diff
@@ -726,8 +728,7 @@ def table_restore(table_name):
         with open(os.path.join(table_dump_path, const_parameters.schema_file), 'r') as f:
             table_schema = json.load(f)
 
-    if 'Table' in table_schema:
-        table_schema = clean_table_schema(table_schema['Table'])
+    table_schema = clean_table_schema(table_schema)
```
Comment (xaf) [inline dynamodb-bnr.py]: Should still check for the 'Table' key in the dictionary for other backups compatibility (if everything was backed up without extracting exactly the tables)
`
if 'Table' in table_schema:
    table_schema = table_schema['Table']
table_schema = clean_table_schema(table_schema)
`

**Review by xaf — CHANGES_REQUESTED** [review]
Thanks for these changes.
In addition to the inline comment, please change:
- Rebase cleanly, your pull requests should not contain a "Merge pull request #1 from XaF/master" commit, but be based on the master of this repository
- Add your signoff to your commits (git commit -s)

Once done, I'll merge this pull request.
Thanks a lot for your contribution :o)

File: dynamodb-bnr.py (hunk 1 above)
Comment (jacekszubert) [reply dynamodb-bnr.py]: Right

**Comment by xaf** [comment]
Merged in commit 8db46751da67a633403c8f16ac42e1b8b4d9a3df
Thanks

**Pull request closed without merging**